gateway-ha/src/test/resources/test-config-template.yml

```diff
@@ -11,6 +11,17 @@ dataStore:
 
 modules:
   - io.trino.gateway.ha.module.HaGatewayProviderModule
+  - io.trino.gateway.ha.module.ClusterStateListenerModule
+  - io.trino.gateway.ha.module.ClusterStatsMonitorModule
+
+managedApps:
+  - io.trino.gateway.ha.clustermonitor.ActiveClusterMonitor
+
+clusterStatsConfiguration:
+  monitorType: INFO_API
+
+monitor:
+  taskDelaySeconds: 1
```

Comment:
could we look at using Airlift duration instead?

Reply:
can I put this in a separate PR? This will require some code changes

Reply:
Sure

Reply:
This config has been there for a while now so it will be a breaking change for this or next release.

Reply:
Okay .. then it should definitely be a separate change. Please file an issue for that work so we dont forget.

Reply:
created https://github.com/trinodb/trino-gateway/issues/520